lib/apartment.rb

```diff
@@ -0,0 +1,28 @@
+class Apartment
+  attr_reader :apartment_specs,
+              :renter
+
+  def initialize(apartment_specs)
+    @apartment_specs = apartment_specs
```

Comment:
This works and I see why you did it this way, but I would pull out each key and value from the hash being passed in here and use instance variables.  

```
def initialize(apartment_specs)
  @number = apartment_specs[:number]
end
```

And so on. Hopefully that is making sense. That way we can just create an `attr_reader` instead of manually creating all of these methods.